### ISSUE TITLE
Multiple choice setting not working.

### ISSUE THREAD

**Issue by markermate** [EyalAr/japakana#1]
When using Firefox and Internet Explorer the "Multiple Choice" selection in the settings page stays as Text Entry.

Steps to reproduce:
Open Settings
Choose "Multiple Choice" checkbox
Switch to Practice

**Comment by EyalAr** [comment]
Yep, haven't yet had the time to implement it. Forgot to remove the option from the settings page :)
